go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/boltdb/bolt v1.3.1
 	github.com/cheggaaa/pb v1.0.5-0.20160713104425-73ae1d68fe0b
 	github.com/djherbis/atime v1.0.0 // indirect
+	github.com/docker/docker v0.0.0-20170502054910-90d35abf7b35
```

Comment:
do we need this change?

Reply:
https://build.ops.storj.io/blue/organizations/jenkins/storj/detail/PR-2891/10/pipeline/43

Reply:
Removed docker.